engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -401,6 +404,75 @@ public String spawnBlock(@Sender EntityRef sender, @CommandParam("blockName") St
         return "Spawned block.";
     }
 
+
+
+	//Works on windows only , i don't know if you can even play the game on other platforms
```

Comment:
Yep, Terasology's crossplatform. `ping` is a UNIX utility but its' output may be not identical to the Windows implementation - I'd use an alternative method of retrieving ping time (like sockets - thanks for the suggestion @portokaliu!)